src/test/cpp/gtester/ParsePopulationModel.cpp

```diff
@@ -10,7 +10,7 @@ namespace Tests {
 
 TEST(ParsePopulationModel, ParseDefaultPopulationModel)
 {
-	std::ifstream pop_file{"data/population_model_default.xml"};
+	std::ifstream pop_file{"../data/population_model_default.xml"};
```

Comment:
Oh, maybe we can let CMake put the project’s install directory in a `-D` flag, and base our paths off that? Sounds like a fuss. Not sure how this kind of thing is generally handled.